de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/ServerConfig.java

```diff
@@ -9,6 +9,7 @@
     private static final String JID_KEY = "de.fu_berlin.inf.dpp.server.jid";
     private static final String PASSWORD_KEY = "de.fu_berlin.inf.dpp.server.password";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'PASSWORD' detected in this expression, review this potentially hardcoded credential. ([squid:S2068](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=143&issues=AWSoM9Lil6vpliCepjg0&open=AWSoM9Lil6vpliCepjg0)

[ISSUE_KEY]: AWSoM9Lil6vpliCepjg0